community/kernel/src/main/java/org/neo4j/kernel/impl/query/Neo4jTransactionalContext.java

```diff
@@ -245,12 +234,6 @@ public Statement statement()
     }
 
     @Override
-    public void check()
```

Comment:
I just removed it because it was not used. If we want it for going forward I can try to bring it back.